Makefile

```diff
@@ -8,11 +10,21 @@ build:
 format:
 	find . -name '*.go' -exec gofmt -s -w {} +
 
-.PHONY: check
-check:
+.PHONY: ci-like-lint
```

Comment:
Probably out of the scope of this PR but here goes nothing: I was always taught that it was a "best practice" (at least in the BSD ecosystem) to regroup the `.PHONY` rules into a single one at the end of the *Makefile*: `.PHONY: ci-like-lint lint tests etc...`. I have no strong opinions about this, just wanted to make sure to comment to find out if anyone does have opinions about it 🐻

Reply:
Just for the record, I'll stay with @oleiade here.

The reason is since that suggestion also makes the file more readable (the property of the code that I value since mostly we read code than write). 

The targets in Makefile are rare to change (after some point), but reading of the Makefile to understand it is a more typical action :)